Library/include/CSP/Systems/EventTicketing/EventTicketing.h

```diff
@@ -26,6 +26,13 @@ enum class EventTicketingVendor
 	Unknown
 };
 
+enum class TicketStatus
+{
+	Purchased = 0,
```

Comment:
I'm not sure where the best place to leave this documentation is, but writing down somewhere what 'purchased' means versus 'redeemed' would be awesome, as its not clear to me as the reader what the difference is/when I should expect one to be returned over the other.

Reply:
My understanding is that for the Submit, we should only expect to see redeemed. I think it would be for another API endpoint that we could see a purchased value but that's not something we have yet. However, it is part of the API that purchased can be a value so we have to cover it. 

My guess is that purchased means we do have knowledge of the ticket but a user has not actually submitted it for redemption yet. 

I think this sort of nuanced documentation probably belongs at source in CHS docs rather than in CSP where we are only wrapping their interpretations which could change over time.

Reply:
I think there is still value in documenting this in the public API to some degree (at your discretion) - because it is valuable information for those seeking to build their own services that interact with CSP. For those people, knowing how CSP expects to interact with services and how it articulates the data it gets back via the API is important context.